docs/tutorials/index.md

```diff
@@ -7,48 +7,19 @@ type: markdown
-
- -  Internal site text link
- -  External link
-
-### [Section 2]
-[Body Copy] Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed blandit consequat finibus. Nullam ullamcorper ex sit amet laoreet mollis. Morbi quis porttitor sapein. Mauris malesuada libero a egestas iaculis.
-
-```ruby
-// Create a default SSL ChannelCredentials object.
-auto channel_creds = grpc::SslCredentials(grpc::SslCredentialsOptions()); 
-// Create a channel using the credentials created in the previous step.
-auto channel = grpc::CreateChannel(server_name, creds);
-// Create a stub on the channel.
-std::unique_ptr<Greeter::Stub> stub(Greeter::NewStub(channel));
-// Make actual RPC calls on the stub.
-grpc::Status s = stub->sayHello(&context, *request, response);
-```
-
-#### [Section 2.1]
-[Body Copy]Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed blandit consequat finibus. Nullam ullamcorper ex sit amet laoreet mollis. Morbi quis porttitor sapein. Mauris malesuada libero a egestas iaculis. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed blandit consequat finibus. Nullam ullamcorper ex sit amet laoreet mollis. Morbi quis porttitor sapein. Mauris malesuada libero a egestas iaculis.
-
-#### [Section 2.2]
-[Body Copy]Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed blandit consequat finibus. Nullam ullamcorper ex sit amet laoreet mollis. Morbi quis porttitor sapein. Mauris malesuada libero a egestas iaculis. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed blandit consequat finibus. Nullam ullamcorper ex sit amet laoreet mollis. Morbi quis porttitor sapein. Mauris malesuada libero a egestas iaculis.
-
-
-### [Section 3]
-[Body Copy] Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed blandit consequat finibus. Nullam ullamcorper ex sit amet laoreet mollis. Morbi quis porttitor sapein. Mauris malesuada libero a egestas iaculis.
-
-#### [Section 3.1]
-[Body Copy]Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed blandit consequat finibus. Nullam ullamcorper ex sit amet laoreet mollis. Morbi quis porttitor sapein. Mauris malesuada libero a egestas iaculis. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed blandit consequat finibus. Nullam ullamcorper ex sit amet laoreet mollis. Morbi quis porttitor sapein. Mauris malesuada libero a egestas iaculis.
+These pages contain a collection of tutorials on the gRPC API and the kinds of RPCs that are available.
+
+ - [C++](basic/c.html)
+ - [Java](basic/java.html)
+ - [Python](basic/python.html)
+ - [Go](basic/go.html)
+ - [Ruby](basic/ruby.html)
+ - [Node.js](basic/node.html)
+ - [C#](basic/csharp.html)
+ - [Objective-C](basic/objective-c.html)
+ - [PHP](basic/php.html)
+
+We also have a growing number of tutorials on a few specific topics.
```

Comment:
We also have a growing number of tutorials on follow-on topics, with more in the pipeline.